__tests__/app.test.js

```diff
@@ -319,6 +319,56 @@ describe("GET /api/users", () => {
   });
 });
 
+describe("GET /api/articles/sorting_queries", () => {
+  test("200: accepts a sort_by query which sorts by one key (descending order by default)", () => {
+    return request(app)
+      .get("/api/articles?sort_by=topic")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy("topic", { descending: true });
+      });
+  });
+  test("200: accepts an order query (key created_at as default)", () => {
+    return request(app)
+      .get("/api/articles?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy("created_at", { ascending: true });
```

Comment:
Please read the documentation for jest-sorted. ascending is not a valid option for toBeSortedBy. It only works because the default is descending false.